egs/vivos/asr1/RESULTS.md

```diff
@@ -0,0 +1,123 @@
+# pytorch Transducer (enc: VGG2L + 6 x Transformer, dec: 1 x LSTM)
+
+- Environments:
+  - date: `Sun Feb  9 20:50:30 CET 2020`
+  - python version: `3.7.3 (default, Mar 27 2019, 22:11:17)  [GCC 7.3.0]`
+  - espnet version: `espnet 0.6.1`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.0.1.post2`
+  - Git hash: `b19ae06fa0ee36d9a65ca701dd169d52cf1a24fd`
+  - Commit date: `Fri Feb 7 16:32:23 2020 +0100
+
+- Results (obtained with `$ show_results.sh`)
+
+```
+# CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_test_decode_transducer|760|32771|85.3|11.0|3.7|3.0|17.7|93.6|
+|decode_test_decode_transducer_lm|760|32771|85.7|10.5|3.8|2.9|17.2|93.3|
+|decode_test_decode_transducer_lm_word7184|760|32771|86.1|10.2|3.7|2.7|16.7|91.8|
+|decode_train_dev_decode_transducer|100|4007|83.8|13.6|2.6|3.3|19.5|98.0|
+|decode_train_dev_decode_transducer_lm|100|4007|84.4|12.9|2.7|2.8|18.4|97.0|
+|decode_train_dev_decode_transducer_lm_word7184|100|4007|84.9|12.2|2.9|2.8|17.9|97.0|
+
+# WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_test_decode_transducer|760|7722|63.5|35.6|0.9|0.5|37.0|93.6|
+|decode_test_decode_transducer_lm|760|7722|65.0|33.8|1.2|0.4|35.5|93.3|
+|decode_test_decode_transducer_lm_word7184|760|7722|66.5|32.3|1.2|0.4|33.9|91.8|
+|decode_train_dev_decode_transducer|100|927|60.7|39.1|0.2|0.2|39.5|98.0|
+|decode_train_dev_decode_transducer_lm|100|927|62.5|37.3|0.2|0.0|37.5|97.0|
+|decode_train_dev_decode_transducer_lm_word7184|100|927|63.1|36.5|0.4|0.0|36.9|97.0|
+```
+
+# pytorch Transducer (enc: VGG2L + 8 x Transformer, dec: 2 x Transformer)
+
+- Environments
+  - date: `Sun Feb  9 20:50:30 CET 2020`
+  - python version: `3.7.3 (default, Mar 27 2019, 22:11:17)  [GCC 7.3.0]`
+  - espnet version: `espnet 0.6.1`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.0.1.post2`
+  - Git hash: `b19ae06fa0ee36d9a65ca701dd169d52cf1a24fd`
+  - Commit date: `Fri Feb 7 16:32:23 2020 +0100
```

Comment:
If you can upload the pretrained model later, that would be super great. However, it is not necessary.

Reply:
Sure! Which model(s) should I upload? RNN-Transducer, Transformer-Transducer or Transformer/RNN-Transducer? :)